cmd/all-in-one/all_in_one_test.go

```diff
@@ -138,9 +138,9 @@ func healthCheck(t *testing.T) {
 	)
 }
 
-func faviconCheck(t *testing.T) {
+func jaegerLogoCheck(t *testing.T) {
 	t.Log("Checking favicon...")
-	resp, err := http.Get(queryURL + "/favicon.ico")
+	resp, err := http.Get(queryURL + "/static/jaeger-logo-ab11f618.svg")
```

Comment:
I wonder if the hash in the filename may change over time and break this test. If it's based on the file checksum then it should be stable. We can improve the test later, e.g. request home page first, and look for logo URL in HTML.

Reply:
the hash should not change if the logo does not change